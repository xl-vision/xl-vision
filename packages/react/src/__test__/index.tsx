describe(`check package '@xl-vision/react' entry`, () => {
  it('list components', () => {
    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,global-require
    const components = require('..');
    expect(Object.keys(components)).toMatchSnapshot();
  });
});
