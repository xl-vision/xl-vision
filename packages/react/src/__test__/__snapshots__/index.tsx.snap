// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check package '@xl-vision/react' entry list components 1`] = `
Array [
  "styled",
  "createGlobalStyles",
  "ThemeProvider",
  "Transition",
  "CSSTransition",
  "CollapseTransition",
  "TransitionGroup",
  "Row",
]
`;
